### DISCUSSION TITLE
Get site ID

### DISCUSSION THREAD

**Discussion by trentwiles** [plausible/analytics#1616] [Self-Hosted Support] [answered]
Sorry. might be a silly question, but I'm trying to get the ID for my plausible site. I'm self hosting. Where could I find this info? This is my API request so far:

```
$ curl https://mysite/api/v1/stats/realtime/visitors -H "Authorization: Bearer abcdefg"
{"error":"Missing site ID. Please provide the required site_id parameter with your request."}
```

**Comment by ukutaht** [comment, marked as answer]
The site id is the same thing as the `domain` field in your Plausible dashboard.

Sorry about the confusing naming. We're planning to move to numeric site IDs so in the future there probably will be a separate `site_id` field but for now we just use the domain name.
